app/assets/javascripts/discourse/app/components/edit-category-security.hbs

```diff
@@ -15,11 +15,12 @@
           <span class="cell">{{i18n "category.permissions.see"}}</span>
           <span class="cell">{{i18n "category.permissions.reply"}}</span>
           <span class="cell">{{i18n "category.permissions.create"}}</span>
+          <span class="cell"></span>
```

Comment:
Maybe to avoid an empty tag we can make the last cell double width? Something like:
```css
.cell:last-child {
width: 40%;
}
```
and to move the trash button to the right, something like: 
```css
{
  .remove-permission {
    margin-left: auto;
  }
}
```

Reply:
Yeah I think generally the empty cell is a better way to handle these button columns